src/Umbraco.Deploy.Contrib.Connectors/Umbraco.Deploy.Contrib.Connectors.csproj

```diff
@@ -5,10 +5,7 @@
         <RootNamespace>Umbraco.Deploy.Contrib.Connectors</RootNamespace>
         <TargetFramework>net472</TargetFramework>
     </PropertyGroup>
-
     <ItemGroup>
-        <PackageReference Include="UmbracoDeploy.Core">
-            <Version>4.1.2</Version>
-        </PackageReference>
+      <ProjectReference Include="..\..\..\Deploy-V4\src\Umbraco.Deploy\Umbraco.Deploy.csproj" />
```

Comment:
This should be updated to the package reference again when the new minor is released!

Reply:
Oops, yes, will revert.  We'll need to bump the dependency too.